dino.icu.yaml

```diff
@@ -417,3 +417,8 @@ xn--ct9h:
 xn--rl8h:
   type: CNAME
   value: maxwofford.github.io.
+
+node:
+- ttl: 600
+  type: CNAME
+  value: tgh75972.github.io.
```

Comment:
It doesn't look like there's a github pages site here. Have you published one yet?

Reply:
I have, it's at tgh75972.github.io/personal-website